client/components/paragraphs/index.tsx

```diff
@@ -0,0 +1,31 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+interface ParagraphsProps {
+	children?: any[];
+}
+
+/**
+ * Series of paragraphs rendered from an array of strings.
+ *
+ * @param {Array} Strings to render as separate paragraphs.
+ *
+ * @return	{Array} Paragraph elements.
+ */
+const Paragraphs: React.FC< ParagraphsProps > = ( {
```

Comment:
`React.FC` is not consistent with rest of migration. Is there specigfic reason to use it here?

Reply:
Was added to accommodate different cases with adding children nodes.

Reply:
Sorry, used my wife's laptop. This is indeed intentional and for better dealing with different nested elements.